libraries/AP_Proximity/AP_Proximity_RPLidarA2.cpp

```diff
@@ -76,7 +104,7 @@ AP_Proximity_RPLidarA2::AP_Proximity_RPLidarA2(AP_Proximity &_frontend,
 {
     _uart = serial_manager.find_serial(AP_SerialManager::SerialProtocol_Lidar360, 0);
     if (_uart != nullptr) {
-        _uart->begin(serial_manager.find_baudrate(AP_SerialManager::SerialProtocol_Lidar360, 0));
+        _uart->begin(serial_manager.find_baudrate(AP_SerialManager::SerialProtocol_Lidar360, 0), RX_BUFFER_SIZE, 256);
```

Comment:
The A2 series is 115200 bps.

Reply:
The baud rate is set in a parameter: SERIALn_BAUD for the whole port, nothing to do with the code itself